Makefile

```
@@ -3,7 +3,7 @@ USERID=$(shell id -u)
 GROUPID=$(shell id -g)
 
 # API Spec branch name
-BRANCH := master
+BRANCH := spec-auto-scaling
```

```suggestion
BRANCH := master
```

To be applied when https://github.com/giantswarm/api-spec/pull/121 is merged